pkgs/by-name/kc/kclvm_cli/package.nix

```diff
@@ -1,30 +1,46 @@
 { lib
+, stdenv
 , rustPlatform
 , fetchFromGitHub
 , kclvm
+, darwin
+, rustc
+,
 }:
 rustPlatform.buildRustPackage rec {
   pname = "kclvm_cli";
-  version = "0.8.7";
+  version = "0.9.3";
 
   src = fetchFromGitHub {
     owner = "kcl-lang";
     repo = "kcl";
     rev = "v${version}";
-    hash = "sha256-ieGpuNkzT6AODZYUcEanb7Jpb+PXclnQ9KkdmlehK0o=";
+    hash = "sha256-nk5oJRTBRj0LE2URJqno8AoZ+/342C2tEt8d6k2MAc8=";
   };
 
   sourceRoot = "source/cli";
-  cargoLock.lockFile = ./Cargo.lock;
+
+  cargoLock = {
```

Comment:
You can delete `cargoLock` and regenerate `cargoHash` in this case.

Reply:
Also delete `Cargo.lock`

Reply:
@Aleksanaa Done :)